test/CMakeLists.txt

```diff
@@ -44,6 +44,31 @@ OPTION(CROSS_COMPILE_ARM "cross compile for ARM targets" OFF)
 set(TARGET_ARCH "native" CACHE STRING "Target architecture arguments")
 
 if (CMAKE_CXX_COMPILER_ID MATCHES "Clang" OR CMAKE_CXX_COMPILER_ID MATCHES "GNU" OR CMAKE_CXX_COMPILER_ID MATCHES "Intel")
+    if (NOT WIN32 AND NOT ANDROID)
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wunused-parameter -Wextra -Wreorder")
+        # Users may override the c++ standard:
+        if(NOT DEFINED CMAKE_CXX_STANDARD OR "${CMAKE_CXX_STANDARD}" STREQUAL "")
+            if (ENABLE_XTL_COMPLEX)
+                CHECK_CXX_COMPILER_FLAG("-std=c++14" HAS_CPP14_FLAG)
+                if (NOT HAS_CPP14_FLAG)
+                    message(FATAL_ERROR "Unsupported compiler -- xsimd requires C++14 support when xtl complex support is enabled")
+                endif()
+                set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
+            else()
+                CHECK_CXX_COMPILER_FLAG("-std=c++11" HAS_CPP11_FLAG)
+                if (NOT HAS_CPP11_FLAG)
+                    message(FATAL_ERROR "Unsupported compiler -- xsimd requires C++11 support!")
+                else()
+                    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+                endif()
+            endif()
+        endif()
+    endif()
+
+    if (NOT CROSS_COMPILE_ARM)
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -fPIC") 
```

Comment:
This should be part of the "not WIN32" branch, otherwise the build on Windows with clang-cl fails.

Reply:
Should be fixed with f8ddeee